Rakefile

```diff
@@ -50,17 +50,17 @@ namespace :crowbar do
     ).flatten.compact.uniq
   end
 
-  desc "Init all barclamps"
+  desc "Initialize all barclamps (:fork, :clone, :add_upstream, :add_susecloud)"
   task init: [:fork, :clone, :add_upstream, :add_susecloud] do
     # nothing to do here
   end
 
-  desc "Update all clones"
+  desc "Update all clones (:pull_upstream)"
   task update: [:pull_upstream] do
```

Comment:
Actually I suggest just dropping this task.  As explained, use of the word `update` is just confusing here.